src/App.jsx

```diff
@@ -14,45 +15,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FILD_BY_ALPHABET = 'SORT_FILD_BY_ALPHABET';
+const SORT_BY_LENGTH = 'SORT_BY_LENGTH';
+
+function getPrepeadGoods(goods, sortFiled, reverse) {
+  const prepearedGoods = [...goods];
+
+  if (sortFiled) {
+    prepearedGoods.sort((good1, good2) => {
+      switch (sortFiled) {
```

Comment:
```suggestion
      switch (sortField) {
```
I guess you meant field. Or better call it `sortType`